apps/dialtone-documentation/docs/guides/contributing/index.md

```diff
@@ -19,12 +19,16 @@ in our Dialtone repository.
 
 If you need to add an icon into Dialtone, here’s how you would go about doing that.
 
-### For system icons
+### For system icons and illustrations
+
+This process is similar for both icons and illustrations.
+The exported SVG file(s) should be placed in the appropriate folder category inside `packages/dialtone-icons/src/svg/illustrations/`.
+The keywords for the illustrations should be added to the `keywords-illustrations.json` file.
 
 1. Create a new branch starting with "feat/" in the name.
-2. Place the exported SVG file(s) in the appropriate folder category inside `packages/dialtone-icons/src/svg/`
+2. Place the exported SVG file(s) in the appropriate folder category inside `packages/dialtone-icons/src/svg/icons/`
 3. If you need to add keywords related to the icon.
-   - Add the icon name to `packages/dialtone-icons/src/keywords.json` into the correct category.
+   - Add the icon name to `packages/dialtone-icons/src/keywords-icons.json` into the correct category.
```

Comment:
Split this into two separate groups of steps, one for icons and one for illustrations. I know it seems a little repetitive but when someone is adding an icon for the first time they're going to want exact steps to easily follow. Right now they kind of have to figure out on their own what to change when adding an illustration

Reply:
Ye I thought that the paragraph at the top would be enough but ok, lemme added it twice.